### ISSUE TITLE
A11y: Missing visual indicator of audio activity

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5224]
### Finding Information
- **Checkpoint ID:** 6.5.5
- **View/Component:** Start a call

### Description
When a call is started, audio activity is not indicated visually.

### Recommended Action
Provide a simple visual cue, such as a dot/LED or another on/off indicator that flickers to show activity.
